### ISSUE TITLE
Is it possible to navigate to next omni with Tab?

### ISSUE THREAD

**Issue by powerman** [ervandew/supertab#152]
When I start omni-completion with Tab only was to navigate to next variant is using Ctrl-N. Shift-Tab works ok (similar to Ctrl-P). But "Tab Tab" sequence works like "Ctrl-X Ctrl-O Ctrl-X Ctrl-O".
At glance look in sources it looks like pumvisible() expected to return true while user is it Ctrl-X menu, but this doesn't work (it returns false, maybe because there are no real "menu" on screen, just "--- Omni… (^O^N^P)…" line at bottom of screen).

**Comment by powerman** [comment]
Yeah, looks like my guess was correct: with `set completeopt=menu` everything works fine. So, is it possible to make it work with empty `&completeopt`?

**Comment by ervandew** [comment]
Give the `supertab_completion_mode` branch a try. I haven't done extensive testing on it, but it should do the trick as long as you are using a version of vim that has the `CompleteDone` autocmd event (the following should echo 1: `:echo exists('##CompleteDone')`).

Let me know if you run into any issues.

**Comment by powerman** [comment]
Cool, thanks! Works for me.
